src/lib/transaction_database/intf.ml

```diff
@@ -0,0 +1,34 @@
+open Coda_base
+open Signature_lib
+
+module type S = sig
```

Comment:
can you add some comments to a few of the functions here? Questions I'd have if I saw this signature include: Is the `Transaction.t` I pass to `get_earlier_transactions` inclusive or exclusive of the earlier one? What do the `get_*` functions do with the public key?

Reply:
Thanks for this. I was able to interface simpler for some methods